### DISCUSSION TITLE
Search bar for the error logs.

### DISCUSSION THREAD

**Discussion by Jdance-Media** [SmartlyDressedGames/Unturned-3.x-Community#3938] [Ideas]
I got a lot of errors (some from other mods). Would be cool to see a checkbox to filter out errors for workshop mods 👉👈.

**Comment by DanielWillett** [comment]
Would be nice, and/or a way to filter by type and mod, similar to GitHub issue tags.

**Comment by GazziFX** [comment]
`-NoWorkshopSubscriptions` Disable loading of all Steam Workshop subscriptions. This can be helpful when troubleshooting issues.

**Reply by Jdance-Media** [reply]
Great in many cases, but I like to have certain workshop mods loaded on purpose, especially when making my own mods that depend on those mods.
